src/BodyRow.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+
+import "./BodyRow.css";
+//import Messages from "./Messages";
+
+const BodyRow = ({ content, messages,setMessage}) => {
+  const [isSelected, setIsSelected] = useState(false);
+
+  const handleClick = () => {
+    setIsSelected(!isSelected);
+  };
+//   useEffect(() => {
+//     //fetch(`https://cyf-react.glitch.me/er`)
+//     //fetch(`https://cyf-react.glitch.me/delayed`)
+//     fetch(`https://cyf-react.glitch.me`)
+//       .then(res => res.json())
+//       .then(data => setBooking(data))
+//       .catch(function(e) {
+//         setError(true);
+//         console.log(e.message);
+//       });
+//   }, []);
```

Comment:
**Style**

Please remove comments from code when not required or needed. Helps keep code clean and easier to read